stdlib/public/runtime/SwiftValue.mm

```diff
@@ -458,6 +464,11 @@ - (NSUInteger)hash {
 	    selfHeader->type, hashableConformance);
   }
 
+  if (!runtime::bincompat::useLegacySwiftObjCHashing()) {
```

Comment:
Darn.  That `!` should not have been there...   Follow-up PR will fix this.

Reply:
Ouch. Lots of trouble from a line and a dot.